test/resolvers/customer.test.js

```diff
@@ -8,6 +8,9 @@ const authorisationOrganisation = sitiAgriAuthorisationOrganisation({ organisati
 const personId = authorisationOrganisation.data.personRoles[0].personId
 const dataSources = {
   ruralPaymentsPortalApi: {
+    getCustomerByCRN () {
+      return { id: '5007136' }
+    },
```

Comment:
this looks like a customerId rather than CRN?

Reply:
also, we need the crn returned, i believe i removed id?

Reply:
Yeah, it is a `customerId`. It's a mock of the endpoint that returns the customer object from the api, which has both `id` and crn` — we then map the `id` to `customerId` in the gql layer, but not here in the data source